### PR TITLE
Node: Add cosmos chains over IBC

### PR DESCRIPTION
**Pull request by bruce-riley**
This PR adds guardian support for the following cosmos chains over IBC:

- cosmoshub 
- evmos
- kujira
- celestia
- osmosis

Support for these chains was previously added to the SDK. They are defined here:
https://github.com/wormhole-foundation/wormhole/blob/main/sdk/js/src/utils/consts.ts#L36-L39

### CODE REVIEW

File: node/pkg/watchers/ibc/watcher.go [hunk 1]
```diff
@@ -63,7 +63,13 @@ func setFeatures(f string) {
 
 var (
 	// Chains defines the list of chains to be monitored by IBC. Add new chains here as necessary.
-	Chains = []vaa.ChainID{vaa.ChainIDSei}
+	Chains = []vaa.ChainID{
+		vaa.ChainIDSei,
+		vaa.ChainIDCosmoshub,
```
Comment (SEJeff) [inline node/pkg/watchers/ibc/watcher.go:68]: We should change this from Cosmos Hub to Cosmos. Hub is the name of the mainnet and gaia is the name of the testnet. Is it possible to change the sdk at this point still?

Unless we want to add a CosmosGaia as well, but that doesn't fit how most all of the rest of the chains are.

**Review by SEJeff — COMMENTED** [review]
The change is totally reasonable, but I feel naming it cosmos hub after the mainnet network explicitly is poor form. We should change it to just `cosmos` like the rest of the networks.

File: node/pkg/watchers/ibc/watcher.go (hunk 1 above)
Comment (evan-gray) [reply node/pkg/watchers/ibc/watcher.go:68]: @SEJeff not sure your comment is quite correct. It looks to me like both [Mainnet](https://github.com/cosmos/mainnet) and [Testnets](https://github.com/cosmos/testnets) are referred to as Cosmos Hub and [Gaia](https://hub.cosmos.network/main/getting-started/what-is-gaia.html) is the Cosmos SDK application.

File: node/pkg/watchers/ibc/watcher.go [hunk 2]
```diff
@@ -63,7 +63,13 @@ func setFeatures(f string) {
 
 var (
 	// Chains defines the list of chains to be monitored by IBC. Add new chains here as necessary.
-	Chains = []vaa.ChainID{vaa.ChainIDSei}
+	Chains = []vaa.ChainID{
```
Comment (evan-gray) [inline node/pkg/watchers/ibc/watcher.go:66]: Just want to clarify that with this change, these chains will be able to emit messages as soon as the channel is allow-listed with governance - is that the idea?

**Review by evan-gray — APPROVED** [review]

File: node/pkg/watchers/ibc/watcher.go (hunk 1 above)
Comment (SEJeff) [reply node/pkg/watchers/ibc/watcher.go:68]: I stand corrected sir. They've changed this from when I last worked on cosmos. Perfect!

**Review by SEJeff — DISMISSED** [review]

File: node/pkg/watchers/ibc/watcher.go (hunk 2 above)
Comment (bruce-riley) [reply node/pkg/watchers/ibc/watcher.go:66]: Yes. When the watcher receives a message for a channel it does not know about, it queries the ibc-receiver contract for the channel to chain mapping. If for some reason, the contract doesn't have the channel to chain mapping (meaning governance hasn't happened yet, but if governance hasn't happened yet, the contract shouldn't publish the message), the watcher generates an error message like "received message from unknown IBC channel" and drops the message.

**Review by evan-gray — APPROVED** [review]

**Review by nik-suri — APPROVED** [review]
LGTM, thanks!

Only nit thought is we might as well as Osmosis (chain id 20) in here as well, since it was defined in the SDK a long time ago and isn't in the guardian codebase.

**Review by nik-suri — APPROVED** [review]
LGTM

**Review by evan-gray — APPROVED** [review]

**Review by SEJeff — APPROVED** [review]

**Pull request merged**